packages/compiler-sfc/__tests__/__snapshots__/compileScriptPropsTransform.spec.ts.snap

```diff
@@ -6,8 +6,8 @@ exports[`sfc props transform $$() escape 1`] = `
 export default {
   props: ['foo'],
   setup(__props) {
-const __props_bar = _toRef(__props, 'bar')
-const __props_foo = _toRef(__props, 'foo')
+const __props_bar = _toRef(__props, 'bar');
```

Comment:
i think you should revert all the `semi` `;` added

Reply:
It's on purpose. That's why I make such a PR. Adds semicolon to avoid ambiguity.

Reply:
Yeah yeah.